### DISCUSSION TITLE
quasar slow button and accordion click performance perception

### DISCUSSION THREAD

**Discussion by abulka** [quasarframework/quasar#9836] [General - Components / Directives / etc]
Clicks on quasar buttons and expansion item components seems laggy to me. E.g. https://quasar.dev/vue-components/button - compared to browser native buttons or even, say fomantic buttons https://fomantic-ui.com/elements/button.html
On my iPad Pro, and on my fast desktop Chrome based browser - its the same.

Quasar [Buttons Toggles](https://quasar.dev/vue-components/button-toggle) seem ok and fast. Is there a setting that might speed up the responsiveness of regular buttons and expansion item components?  I was ready to jump on board the Quasar framework but just cannot get past this issue.

**Comment by metalsadman** [comment]
@abulka  for buttons, might be the ripple order effect making it look slow to you, you can supply an object to q-btn's `ripple` prop to fire it early ie. `<q-btn :ripple="{early: true}"...` or turn it off `:ripple="false"`, as for q-expansion-item, there's a `duration` prop you can set, default is 300ms ie. `<q-expansion-item :duration="100"...`. check the api card for the aforementioned props.
